app/helpers/ontology_search_helper.rb

```diff
@@ -0,0 +1,20 @@
+module OntologySearchHelper
+
+  def render_filter(klass, include_blank='', selected_key: nil)
+    name = klass.name.underscore.to_sym
+    render 'shared/ontology_search_filter', {
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.